qemu/tests/block_hotplug.py

```diff
@@ -228,6 +230,8 @@ def unplug_device(vm, get_disk_cmd, device):
                     test.fail("Failed to unplug disks '%s'" % device.get_param("id"))
                 device_list.remove(device)
             controller.unplug(vm.monitor)
+            time.sleep(5)
```

Comment:
Defined variant "pause" can be used here, like time.sleep(pause).

Reply:
updated, thanks